src/controllers/SlackBotController.js

```diff
@@ -12,7 +12,7 @@ const {
   SLACK_URI,
```

Comment:
Is there a reason that we are not using ES6 features?

Reply:
ES6 should be up, but imports go in other way in nodejs